delta-sharing-protocl-api-description.yml

```diff
@@ -0,0 +1,512 @@
+      description: |
+        This is the API for clients to get a table version without any other
+        extra information. The server usually can implement this API
+        effectively. If a client caches information about a shared table
+        locally, it can store the table version and use this cheap API to
+        quickly check whether their cache is stale and they should re-fetch the
+        data.
+      tags:
+        - tables
+        - official
+      parameters: &tableParameters
+        - in: path
+          name: share
+          required: true
+          description: 'Named share for finding the named schema'
+          schema:
+            type: string
+        - in: path
+          name: schema
+          required: true
+          description: 'Named schema for finding the named table'
+          schema:
+            type: string
+        - in: path
+          name: table
+          required: true
+          description: 'Named table'
+          schema:
+            type: string
+      responses:
+        '400':
+          $ref: "#/components/responses/400"
+        '401':
+          $ref: "#/components/responses/401"
+        '403':
+          $ref: "#/components/responses/403"
+        '404':
+          $ref: "#/components/responses/404"
+        '500':
+          $ref: "#/components/responses/500"
+        '200':
+          description: 'The table was found and has a version'
+          headers:
+            Delta-Table-Version:
+              description: 'A long value which represents the current table version'
+              schema:
+                type: integer
+
+  '/shares/{share}/schemas/{schema}/tables/{table}/metadata':
+    get:
+      operationId: 'GetTableMetadata'
+      summary: 'Query the metadata and schema of the given table'
+      tags:
+        - tables
+        - official
+      parameters: *tableParameters
+      responses:
+        '400':
+          $ref: "#/components/responses/400"
+        '401':
+          $ref: "#/components/responses/401"
+        '403':
+          $ref: "#/components/responses/403"
+        '404':
+          $ref: "#/components/responses/404"
+        '500':
+          $ref: "#/components/responses/500"
+        '200':
+          description: |
+            A sequence of JSON strings containing the table protocol
+            and then the table metadata
+          headers:
+            Delta-Table-Version:
+              description: 'A long value which represents the current table version'
+              schema:
+                type: integer
+          content:
+            'application/x-ndjson':
+              schema:
+                $ref: '#/components/schemas/TableMetadataResponse'
+
+
+  '/shares/{share}/schemas/{schema}/tables/{table}/query':
+    post:
+      operationId: 'QueryTable'
+      summary: 'Query the table'
+      tags:
+        - tables
+        - official
+      parameters: *tableParameters
```

Comment:
We need a `queyTableParameters` and add all the supported parameters: https://github.com/delta-io/delta-sharing/blob/main/PROTOCOL.md#request-body

Reply:
It is part of #/components/schemas/QueryRequest, updated with the md content.